src/cares_wrap.cc

```diff
@@ -2065,17 +2075,19 @@ void SetServers(const FunctionCallbackInfo<Value>& args) {
   int err;
 
   for (uint32_t i = 0; i < len; i++) {
-    CHECK(arr->Get(i)->IsArray());
+    CHECK(arr->Get(env->context(), i).ToLocalChecked()->IsArray());
 
-    Local<Array> elm = Local<Array>::Cast(arr->Get(i));
+    Local<Array> elm =
+        Local<Array>::Cast(arr->Get(env->context(), i).ToLocalChecked());
 
-    CHECK(elm->Get(0)->Int32Value());
-    CHECK(elm->Get(1)->IsString());
-    CHECK(elm->Get(2)->Int32Value());
+    CHECK(elm->Get(env->context(), 0).ToLocalChecked()->Int32Value());
+    CHECK(elm->Get(env->context(), 1).ToLocalChecked()->IsString());
+    CHECK(elm->Get(env->context(), 2).ToLocalChecked()->Int32Value());
 
-    int fam = elm->Get(0)->Int32Value();
-    node::Utf8Value ip(env->isolate(), elm->Get(1));
-    int port = elm->Get(2)->Int32Value();
+    int fam = elm->Get(env->context(), 0).ToLocalChecked()->Int32Value();
+    node::Utf8Value ip(env->isolate(),
+                       elm->Get(env->context(), 1).ToLocalChecked());
+    int port = elm->Get(env->context(), 2).ToLocalChecked()->Int32Value();
```

Comment:
If you want, I think it would make sense to do the same for `Int32Value` as well

Reply:
Good catch. Thanks!